source/server/GameServer.py

```diff
@@ -60,7 +60,13 @@ def Send_publicInfo(self):
 
     def Send_discardInfo(self):
         """Send the update to the discard pile"""
+        # <<<<<<< HEAD
+        # SLW asks which version to keep??  comment out next two lines and keep two below....
+        # info = self.DiscardInfo()
+        # self.SendToAll({"action": "discardInfo", "top_card": info[0].Serialize(), "size": info[1]})
+        # =======
         info = self.discard_info()
```

Comment:
Keep this one here on the bottom - its up to date after the style reformation. I can't edit in this window but you can leave this in place for me to fix in my next MR and/or open a bug case.